examples/nlp/text_classification/conf/text_classification_config.yaml

```diff
@@ -23,44 +23,42 @@ trainer:
   max_steps: null # precedence over max_epochs
   accumulate_grad_batches: 1 # accumulates grads every k batches
   amp_level: O0 # O1/O2 for mixed precision
+  precision: 32 # Should be set to 16 for O1 and O2
```

Comment:
should be 16

Reply:
Is it OK to be 16 when amp_level is O0? I think they have the plan to drop amp_level and keep the precicion?

Reply:
if you do precision 16 and O0 its still O0

Reply:
Updated to 16.